### DISCUSSION TITLE
Adding files to the module

### DISCUSSION THREAD

**Discussion by teciON** [YetiForceCompany/YetiForceCRM#15710] [How to] [answered]
Is it possible to add zip or pdf files to modules, e.g. fixed assets? Same as in the Documents module (see attached image).

![Screenshot_3](https://user-images.githubusercontent.com/99530823/153703890-15ef1a00-c628-48f2-8571-4eb320b1141d.jpg)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
only image files or document relation field

**Reply by teciON** [reply]
Thank you for quick answer.
Do you have any plans to introduce adding files to modules in the near future? If the system got such a function, it would be practically unrivaled.

**Reply by mariuszkrzaczkowski** [reply]
Yes, I have plans but I don't know when it will be added to the system
